### ISSUE TITLE
Spacing under staff

### ISSUE THREAD

**Issue by dedan** [opensheetmusicdisplay/opensheetmusicdisplay#643]
Hey OSMD team,

a happy new year and thanks for this amazing library!

When I render a score using OSMD I get an unnecessary space under the bass staff. Even though I use `drawingParameters: 'compact'`. Is there any way to remove this spacing?

Please see the screenshot below for an example:

<img width="1109" alt="Screen Shot 2020-01-07 at 20 43 08" src="https://user-images.githubusercontent.com/196905/71924240-e7112900-318e-11ea-8499-b1e73cba4d6d.png">

**Comment by sschmidTU** [comment]
Hey, thanks for the comment and report!
I checked this problem and it's not caused by compact mode,
the problem is that some of our Staffline bounding boxes are way too large.

Before i explain why, let me say that i could not fix it yet, but i provided a workaround that shouldn't cause trouble in most rendering cases (only on develop branch until next release):
```js
osmd.EngravingRules.FixStafflineBoundingBox = true;
// (load/render)
```

This disables adding the bounding boxes of ActivitySymbols to Stafflines.
Their bounding boxes seem to cause some problems:
![image](https://user-images.githubusercontent.com/33069673/72457825-b7ce6d80-37c7-11ea-868a-ab5244a697cc.png)

![image](https://user-images.githubusercontent.com/33069673/72455785-b8fd9b80-37c3-11ea-97d8-b82e26f28704.png)
This only seems to happen when there are two stafflines in a system (e.g. piano).
Unfortunately i haven't been able to find the deeper cause yet.

The Beethoven piece is an interesting example, because it only happens for some stafflines, most of them have a correct bounding box:
<img src="https://user-images.githubusercontent.com/33069673/72455901-eea28480-37c3-11ea-80c0-7b5336c40a7b.png" height="300">

The bounding

**Comment by bneumann** [comment]
I am pretty puzzled about these boxes at the end. It seems I ported them a while back but no idea what they are good for. The see to be some click are of sorts. @matt-uib you know what the StaffLineActivitySymbol class should be doing? They are not doing anything at the moment

**Comment by matt-uib** [comment]
yes they are for designed for activating e.g. playback of a staff-line, but this will all come later and can be commented out currently. but their bounding boxes seem as they shall be and it doesn't look to me as these are the problem here... they only stick out a little bit vertically.

**Comment by sschmidTU** [comment]
This is now fixed without a workaround needed in #1244.

<img width="168" alt="image" src="https://user-images.githubusercontent.com/33069673/193062774-30ea4423-8cca-4050-9ee5-8e2f3cb1871a.png">
